src/lib/src/widget-library/select.component.ts

```diff
@@ -92,4 +96,22 @@ export class SelectComponent implements OnInit {
   updateValue(event) {
     this.jsf.updateValue(this, event.target.value);
   }
+
+  showOption(item: any, layoutNode: any): boolean {
+    if (item.conditionKey !== undefined && layoutNode.options.conditionMap !== undefined) {
```

Comment:
I am just wondering if it is equally correct to write `if(item.conditionKey && layoutNode.options.conditionMap)` skipping the !== undefined. I mean, if `0` and `null` are unacceptable values as well.

Reply:
I am not familiar with what is best practice. If this is the style that goes through the rest of the code in this repo, then by all means, its better to keep it this way (using !== undefined).

Reply:
You are right